app/views/specialist_documents/_publish_form.html.erb

```diff
@@ -1,14 +1,18 @@
-<div class="panel panel-danger">
-  <% if warning %>
+<% unless disabled %>
```

Comment:
It seems weird to allow this partial to be included with a parameter that results in it being a no-op. Would it be better to just not include the partial?